### PR TITLE
Remove all scss stuff

### PR DESCRIPTION
**Pull request by sarah-emerson**
Removing all the scss stuff, because this has in the past caused grief for online students, who are mostly not using scss for their Week 6 and final projects because we don't teach it. I also installed prettier/eslint stuff into package.json. Before changing anything I made a copy of this repo called boilerplate-fullstack-scss for anyone who wants to keep/use the scss stuff.

### CODE REVIEW

File: client/webpack.config.js [hunk 1]
```diff
@@ -1,41 +1,23 @@
...
 module.exports = {
-  entry: ['./client/index.js', './client/styles/index.scss'],
+  entry: ['./client/index.js'],
   output: {
     path: path.join(__dirname, '..', 'server', 'public'),
-    filename: 'bundle.js'
+    filename: 'bundle.js',
   },
   mode: 'development',
-  plugins: [
```
Comment (gerardpaapu) [inline client/webpack.config.js:11]: does this mean we're not bundling our css anymore? was that an intended change

**Comment by sarah-emerson** [comment]
My only intended change was to get rid of scss, and I don't know well enough how it is supposed to work. Sounds like I've been overly trigger-happy? Sounds like I should hunt down a example of well-built code without scss and copy it? 
I learn a lot by making these mistakes and I appreciate the feedback btw.

File: client/webpack.config.js (hunk 1 above)
Comment (sarah-emerson) [reply client/webpack.config.js:11]: I've snooped a couple of other repos (`sweet-as-beers` and `react-to-web-api`) and neither of them have this `MiniCssExtractPlugin` stuff. I've never heard of it so presumably neither have our students. Let's keep it simple. So I'm retroactively making this an intentional change! :-)

Comment (gerardpaapu) [reply client/webpack.config.js:11]: okay cool

**Pull request merged**